controllers/general.controller.js

```diff
@@ -0,0 +1,18 @@
+/*
+
+This file contains all controller functionality that is 
+not major enough to be broken out to a dedicated file
+
+*/
+
+// gets all topics from the DB
+const { fetchTopics } = require("../models/general.model");
+
+
+const getTopics = (request, response) => {
+  fetchTopics().then((topics) => {
+    response.status(200).send({ topics });
+  });
+};
+
```

Comment:
Well named function name, good work.

Great to see you returning the response data on the key of `topics` - this is good as it represents the data being returned. This is a good naming convention.